concepticondata/conceptrelations.tsv

```diff
@@ -477,3 +477,5 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 1205	TONGUE	narrower	2777	PIG TONGUE
 1352	KNIFE	narrower	2772	VEGETABLE KNIFE
 744	EGG	narrower	2779	CHICKEN EGG
+1401	DRINK	narrower	2787	DRINK OR EAT
+1336	EAT	narrower	2787	DRINK OR EAT
```

Comment:
Swap the lines (it's also confusing for me), as this needs to read:

* there is concept DRINK OR EAT and **narrower is** concept DRINK

Was confusing me as well, but @xrotwang said this was common style in the specification we follow.